website/docs/r/org_vdc_template.html.markdown

```diff
@@ -0,0 +1,157 @@
+  * `cpu_speed` - (Required for `AllocationVApp`, `AllocationPool` or `Flex`) Specifies the clock frequency, in MHz, for any virtual CPU that is allocated to a VM. Minimum is 256MHz
+  * `memory_allocated` - (Required for `AllocationPool`, `ReservationPool` or `Flex`) The maximum amount of Memory, in MB, available to the VMs running within the VDC that is instantiated from this template
+  * `memory_limit` - (Required for `AllocationVApp`, `ReservationPool` or `Flex`) The limit amount of Memory, in MB, of the VDC that is instantiated from this template. Minimum is 1024MB. 0 means unlimited
+  * `memory_guaranteed` - (Required for `AllocationVApp`, `AllocationPool` or `Flex`) The percentage of the Memory guaranteed to be available to VMs running within the VDC instantiated from this template
+  * `elasticity` - (Required for `Flex`) True if compute capacity can grow or shrink based on demand
+  * `include_vm_memory_overhead` - (Required for `Flex`) True if the instantiated VDC includes memory overhead into its accounting for admission control
+* `storage_profile` - (Required) A block that defines a storage profile that the VDCs instantiated from this template will use. Must be **at least one**, which has the following properties:
+  * `name` - (Required) Name of Provider VDC storage profile to use for the VDCs instantiated from this template
+  * `default` - (Required) True if this is default storage profile for the VDCs instantiated from this template. Only **one** block should have this set to `true`
+  * `limit` - (Required) Storage limit for the VDCs instantiated from this template, in Megabytes. 0 means unlimited
+* `enable_fast_provisioning` - (Optional) If `true`, the VDCs instantiated from this template will have Fast provisioning enabled. Defaults to `false`
+* `enable_thin_provisioning` - (Optional) If `true`, the VDCs instantiated from this template will have Thin provisioning enabled. Defaults to `false`
+* `edge_gateway` - (Optional) VDCs instantiated from this template will create a new Edge Gateway with the provided setup. Required if any `provider_vdc` block
+  has defined a `gateway_edge_cluster_id`. This **unique** block has the following properties:
+  * `name` - (Required) Name of the Edge Gateway
+  * `description` - (Optional) Description of the Edge Gateway
+  * `ip_allocation_count` - (Optional) Allocated IPs for the Edge Gateway. Defaults to 0
+  * `network_name` - (Required) Name of the network to create with the Edge Gateway
+  * `network_description` - (Optional) Description of the network to create with the Edge Gateway
+  * `network_gateway_cidr` - (Required) CIDR of the Edge Gateway for the created network
+  * `static_ip_pool` - (Required) **One block** with a single IP range (this is a constraint due to a bug in VCD) that have the following properties:
```

Comment:
Heads up for reviewers too.
Confirmed that only one block can be set due to a bug. Should we mention like that in the docs?

Reply:
Yes, but I'd rephrase to something like this: "(this is a constraint as of VCD 10.x.x)"

Reply:
Done